src/registrar/templates/domain_dsdata.html

```diff
@@ -5,12 +5,16 @@
 
 
   {% block breadcrumb %}
-  {% if portfolio %}
   <!-- Navigation breadcrumbs -->
+   {% if portfolio %}
+    {% url 'domains' as url %}
+   {% else %}
+     {% url 'home' as url %}
```

Comment:
So this comment is related to the one below. I'm wondering if this conditional is correctly setting the URL.